src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/SearchCriteria.kt

```diff
@@ -1,25 +1,23 @@
 package uk.gov.justice.digital.hmpps.prisonersearch.services
 
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties
-import io.swagger.annotations.ApiModel
-import io.swagger.annotations.ApiModelProperty
+import io.swagger.v3.oas.annotations.media.Schema
 
-@ApiModel("Search Criteria for Prisoner Search")
-
-@JsonIgnoreProperties( value =  [ "isValid" ])
-data class SearchCriteria (
-  @ApiModelProperty(required = false, value = "Prisoner identifier, one of prisoner number, book number, booking ID or PNC", example = "A1234AA,", position = 1)
-  val prisonerIdentifier : String?,
-  @ApiModelProperty(required = false, value = "First Name", example = "John", position = 2)
-  val firstName : String?,
-  @ApiModelProperty(required = false, value = "Last Name", example = "Smith", position = 3)
+@Schema(description = "Search Criteria for Prisoner Search")
+data class SearchCriteria(
+  @Schema(
+    description = "Prisoner identifier, one of pryes but you don't need to isoner number, book number, booking ID or PNC",
+    example = "A1234AA,"
+  )
+  val prisonerIdentifier: String?,
+  @Schema(description = "First Name", example = "John")
+  val firstName: String?,
+  @Schema(description = "Last Name", example = "Smith")
   val lastName: String?,
-  @ApiModelProperty(required = false, value = "Prison Id", notes = "Prison Id or OUT or TRN", example = "MDI", position = 4)
+  @Schema(description = "Prison Id, Prison Id or OUT or TRN", example = "MDI")
   val prisonId: String? = null,
-  @ApiModelProperty(required = false, value = "Include aliases in search", example = "false", position = 5)
-  val includeAliases : Boolean = false
+  @Schema(description = "Include aliases in search", example = "false", required = false, defaultValue = "false")
```

Comment:
generation not all the way there yet, can't cope with a kotlin field with a default value properly, says that the property is required when we know better